src/spz/spz/forms/__init__.py

```diff
@@ -580,64 +579,34 @@ def get_tag(self):
         return self.tag.data
 
 
-class ExportForm(FlaskForm):
-    """Represents a general export form.
-    """
-
-    format = SelectField(
-        'Format',
-        [validators.DataRequired('Das Format muss angegeben werden')],
-        choices=tables.export_file_formats
-    )
-
-    no_sections = BooleanField(
-        'Alle Kurse in einen Abschnitt schreiben'
-    )
-
-    def get_format(self):
-        return self.format.data
-
-    def sections_wanted(self):
-        return not self.no_sections.data
-
-    def __init__(self, *args, **kwargs):
-        super(ExportForm, self).__init__(*args, **kwargs)
-
+class ExportCourseForm(FlaskForm):
+    """Form for exporting one or multiple courses.
 
-class ExportCourseForm(ExportForm):
-    """Represents a form to select course export options.
+       Export format choices differ, depending on the passed language list (current_user.languages).
+       It might be an option to use the value of 'courses' instead.
     """
 
-    select = SelectMultipleField(
+    courses = SelectMultipleField(
```

Comment:
Would be nice to have a "Deselect all" button for the courses.

Reply:
Wouldn't that be pretty redundant to the reset button?

Reply:
Woops, did not see it.